Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/EncryptFileViewModel.cs

```diff
@@ -48,6 +50,11 @@ public EncryptFileViewModel(IDesignServices services) : base(services)
         /// </summary>
         public DesignInArgument<SecureString> KeySecureString { get; set; } = new DesignInArgument<SecureString>();
 
+        /// <summary>
+        /// Switches between 
```

Comment:
?

Reply:
changed it to 'Switches Key as string or secure string '